etcdctl/ctlv3/command/ep_command.go

```diff
@@ -51,6 +51,7 @@ func newEpStatusCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "status",
 		Short: "status prints out the status of endpoints specified in `--endpoints` flag",
+		Long:  "when 'write-out' is simple, prints comma-separated status lists (endpoint, ID, version, db size, is leader, raft term, raft index)",
```

Comment:
when --write-out is set to simple, this command prints out comma-separated status lists for each endpoint.
The items in the lists are endpoint, ID, version, db size ...